lib/decoratex/schema.ex

```diff
@@ -41,9 +41,11 @@ defmodule Decoratex.Schema do
 
       decorations = Enum.reverse(@decorations)
 
-      Module.eval_quoted(__ENV__, [
-        Schema.__decorations__(decorations)
-      ])
+      Code.eval_quoted(
+      Schema.__decorations__(decorations),
+      [],
+      __ENV__
+      )
```

Comment:
Thx! Please, apply some standard code format like

```suggestion
      Code.eval_quoted(
        Schema.__decorations__(decorations),
        [],
        __ENV__
      )
```

or just in one line

```suggestion
      Code.eval_quoted(Schema.__decorations__(decorations), [], __ENV__)
```

Reply:
Hi @rsierra , 
I used the online editor for this one, as it's a 1 liner
but Elixir 1.18 compiler keeps making noise about this warning.

thank you